package/MDAnalysis/analysis/contacts.py

```diff
@@ -405,6 +405,13 @@ def __init__(self, u, select, refgroup, method="hard_cut", radius=4.5,
              Show detailed progress of the calculation if set to ``True``; the
              default is ``False``.
 
+        Attributes
+        ----------
+        n_initial_contacts : int
+             Total number of initial contacts.
+        r0 : list
```

Comment:
```suggestion
        r0 : list[numpy.ndarray]
```

In Python typing, you can use `[]` to specify what type is inside the container too!